### ISSUE TITLE
webpack is broken with @nextcloud/vue 7.4.0

### ISSUE THREAD

**Issue by serotonie** [SergeyMosin/Appointments#395]
Hi there,

I just cloned the repo and when I try to run `npm run watch` or `npm run build` I get the following trace.

Blocking @nextcloud/vue to 7.3.0 in package.json solved the Issue temporary.
```
$ npm run build

> appointments@0.0.1 build
> webpack --progress --config webpack.prod.js

assets by status 4.21 MiB [cached] 7 assets
orphan modules 1.72 MiB [orphan] 294 modules
runtime modules 1.25 KiB 7 modules
modules by path ./node_modules/ 4.33 MiB 389 modules
modules by path ./src/ 1.57 MiB
  modules by path ./src/components/ 42.8 KiB 32 modules
  modules by path ./src/*.js 1.53 MiB 3 modules
  modules by path ./src/*.vue 1.97 KiB
    ./node_modules/vue-style-loader/index.js!./node_modules/css-loader/dist/cjs.js!./node_modules/vue-loader/lib/loaders/stylePostLoader.js!./node_modules/vue-loader/lib/index.js??vue-loader-options!./src/App.vue?vue&type=style&index=0&id=646e4c13&prod&scoped=true&lang=css& 1.58 KiB [built] [code generated]
    ./node_modules/css-loader/dist/cjs.js!./node_modules/vue-loader/lib/loaders/stylePostLoader.js!./node_modules/vue-loader/lib/index.js??vue-loader-options!./src/App.vue?vue&type=style&index=0&id=646e4c13&prod&scoped=true&lang=css& 402 bytes [built] [code generated]
modules by path ./scss/*.scss 119 bytes
  ./scss/form.scss 59 bytes [built] [code generated]
  ./scss/style.scss 60 bytes [built] [code generated]
modules by path ./css/*.css 8.79 KiB
  ./css/datepicker.css 1.07 KiB [built] [code generated]
  ./node_modules/css-loader/dist/cjs.js!./css/datepicker.css 7.72 KiB [built] [code generated]

ERROR in ./node_modules/@nextcloud/vue/dist/Components/NcActions.js 2:127040-127057
Module not found: Error: Can't resolve 'assert' in '/home/pi/devNextcloud/appointments/node_modules/@nextcloud/vue/dist/Components'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "assert": require.resolve("assert/") }'
        - install 'assert'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "assert": false }
 @ ./node_modules/babel-loader/lib/index.js??clonedRuleSet-4.use!./node_modules/vue-loader/lib/index.js??vue-loader-options!./src/App.vue?vue&type=script&lang=js& 11:0-66 60:13-20
 @ ./src/App.vue?vue&type=script&lang=js& 1:0-178 1:194-197 1:199-374 1:199-374
 @ ./src/App.vue 2:0-55 3:0-50 3:0-50 10:2-8
 @ ./src/main.js 2:0-28 24:13-16

ERROR in ./node_modules/@nextcloud/vue/dist/Components/NcActions.js 2:127203-127218
Module not found: Error: Can't resolve 'util' in '/home/pi/devNextcloud/appointments/node_modules/@nextcloud/vue/dist/Components'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "util": require.resolve("util/") }'
        - install 'util'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "util": false }
 @ ./node_modules/babel-loader/lib/index.js??clonedRuleSet-4.use!./node_modules/vue-loader/lib/index.js??vue-loader-options!./src/App.vue?vue&type=script&lang=js& 11:0-66 60:13-20
 @ ./src/App.vue?vue&type=script&lang=js& 1:0-178 1:194-197 1:199-374 1:199-374
 @ ./src/App.vue 2:0-55 3:0-50 3:0-50 10:2-8
 @ ./src/main.js 2:0-28 24:13-16

ERROR in ./node_modules/@nextcloud/vue/dist/Components/NcAppContent.js 2:115133-115150
Module not found: Error: Can't resolve 'assert' in '/home/pi/devNextcloud/appointments/node_modules/@nextcloud/vue/dist/Components'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "assert": require.resolve("assert/") }'
        - install 'assert'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "assert": false }
 @ ./node_modules/babel-loader/lib/index.js??clonedRuleSet-4.use!./node_modules/vue-loader/lib/index.js??vue-loader-options!./src/App.vue?vue&type=script&lang=js& 7:0-72 43:16-26
 @ ./src/App.vue?vue&type=script&lang=js& 1:0-178 1:194-197 1:199-374 1:199-374
 @ ./src/App.vue 2:0-55 3:0-50 3:0-50 10:2-8
 @ ./src/main.js 2:0-28 24:13-16

ERROR in ./node_modules/@nextcloud/vue/dist/Components/NcAppContent.js 2:115294-115309
Module not found: Error: Can't resolve 'util' in '/home/pi/devNextcloud/appointments/node_modules/@nextcloud/vue/dist/Components'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "util": require.resolve("util/") }'
        - install 'util'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "util": false }
 @ ./node_modules/babel-loader/lib/index.js??clonedRuleSet-4.use!./node_modules/vue-loader/lib/index.js??vue-loader-options!./src/App.vue?vue&type=script&lang=js& 7:0-72 43:16-26
 @ ./src/App.vue?vue&type=script&lang=js& 1:0-178 1:194-197 1:199-374 1:199-374
 @ ./src/App.vue 2:0-55 3:0-50 3:0-50 10:2-8
 @ ./src/main.js 2:0-28 24:13-16

ERROR in ./node_modules/@nextcloud/vue/dist/Components/NcAppNavigation.js 2:117600-117617
Module not found: Error: Can't resolve 'assert' in '/home/pi/devNextcloud/appointments/node_modules/@nextcloud/vue/dist/Components'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "assert": require.resolve("assert/") }'
        - install 'assert'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "assert": false }
 @ ./node_modules/babel-loader/lib/index.js??clonedRuleSet-4.use!./node_modules/vue-loader/lib/index.js??vue-loader-options!./src/App.vue?vue&type=script&lang=js& 6:0-78 42:19-32
 @ ./src/App.vue?vue&type=script&lang=js& 1:0-178 1:194-197 1:199-374 1:199-374
 @ ./src/App.vue 2:0-55 3:0-50 3:0-50 10:2-8
 @ ./src/main.js 2:0-28 24:13-16

ERROR in ./node_modules/@nextcloud/vue/dist/Components/NcAppNavigation.js 2:117708-117723
Module not found: Error: Can't resolve 'util' in '/home/pi/devNextcloud/appointments/node_modules/@nextcloud/vue/dist/Components'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "util": require.resolve("util/") }'
        - install 'util'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "util": false }
 @ ./node_modules/babel-loader/lib/index.js??clonedRuleSet-4.use!./node_modules/vue-loader/lib/index.js??vue-loader-options!./src/App.vue?vue&type=script&lang=js& 6:0-78 42:19-32
 @ ./src/App.vue?vue&type=script&lang=js& 1:0-178 1:194-197 1:199-374 1:199-374
 @ ./src/App.vue 2:0-55 3:0-50 3:0-50 10:2-8
 @ ./src/main.js 2:0-28 24:13-16

ERROR in ./node_modules/@nextcloud/vue/dist/Components/NcAppNavigationItem.js 2:171717-171734
Module not found: Error: Can't resolve 'assert' in '/home/pi/devNextcloud/appointments/node_modules/@nextcloud/vue/dist/Components'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "assert": require.resolve("assert/") }'
        - install 'assert'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "assert": false }
 @ ./node_modules/babel-loader/lib/index.js??clonedRuleSet-4.use!./node_modules/vue-loader/lib/index.js??vue-loader-options!./src/App.vue?vue&type=script&lang=js& 9:0-86 44:23-40
 @ ./src/App.vue?vue&type=script&lang=js& 1:0-178 1:194-197 1:199-374 1:199-374
 @ ./src/App.vue 2:0-55 3:0-50 3:0-50 10:2-8
 @ ./src/main.js 2:0-28 24:13-16

ERROR in ./node_modules/@nextcloud/vue/dist/Components/NcAppNavigationItem.js 2:171880-171895
Module not found: Error: Can't resolve 'util' in '/home/pi/devNextcloud/appointments/node_modules/@nextcloud/vue/dist/Components'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "util": require.resolve("util/") }'
        - install 'util'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "util": false }
 @ ./node_modules/babel-loader/lib/index.js??clonedRuleSet-4.use!./node_modules/vue-loader/lib/index.js??vue-loader-options!./src/App.vue?vue&type=script&lang=js& 9:0-86 44:23-40
 @ ./src/App.vue?vue&type=script&lang=js& 1:0-178 1:194-197 1:199-374 1:199-374
 @ ./src/App.vue 2:0-55 3:0-50 3:0-50 10:2-8
 @ ./src/main.js 2:0-28 24:13-16

ERROR in ./node_modules/@nextcloud/vue/dist/Components/NcAppSettingsDialog.js 2:171977-171994
Module not found: Error: Can't resolve 'assert' in '/home/pi/devNextcloud/appointments/node_modules/@nextcloud/vue/dist/Components'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "assert": require.resolve("assert/") }'
        - install 'assert'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "assert": false }
 @ ./node_modules/babel-loader/lib/index.js??clonedRuleSet-4.use!./node_modules/vue-loader/lib/index.js??vue-loader-options!./src/App.vue?vue&type=script&lang=js& 10:0-86 45:23-40
 @ ./src/App.vue?vue&type=script&lang=js& 1:0-178 1:194-197 1:199-374 1:199-374
 @ ./src/App.vue 2:0-55 3:0-50 3:0-50 10:2-8
 @ ./src/main.js 2:0-28 24:13-16

ERROR in ./node_modules/@nextcloud/vue/dist/Components/NcAppSettingsDialog.js 2:172246-172261
Module not found: Error: Can't resolve 'util' in '/home/pi/devNextcloud/appointments/node_modules/@nextcloud/vue/dist/Components'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "util": require.resolve("util/") }'
        - install 'util'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "util": false }
 @ ./node_modules/babel-loader/lib/index.js??clonedRuleSet-4.use!./node_modules/vue-loader/lib/index.js??vue-loader-options!./src/App.vue?vue&type=script&lang=js& 10:0-86 45:23-40
 @ ./src/App.vue?vue&type=script&lang=js& 1:0-178 1:194-197 1:199-374 1:199-374
 @ ./src/App.vue 2:0-55 3:0-50 3:0-50 10:2-8
 @ ./src/main.js 2:0-28 24:13-16

ERROR in ./node_modules/@nextcloud/vue/dist/index.module.js 2:500677-500694
Module not found: Error: Can't resolve 'assert' in '/home/pi/devNextcloud/appointments/node_modules/@nextcloud/vue/dist'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "assert": require.resolve("assert/") }'
        - install 'assert'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "assert": false }
 @ ./node_modules/babel-loader/lib/index.js??clonedRuleSet-4.use!./node_modules/vue-loader/lib/index.js??vue-loader-options!./src/components/ActionInputExt.vue?vue&type=script&lang=js& 1:0-47 4:19-32 6:11-24 10:6-41
 @ ./src/components/ActionInputExt.vue?vue&type=script&lang=js& 1:0-195 1:211-214 1:216-408 1:216-408
 @ ./src/components/ActionInputExt.vue 2:0-66 3:0-61 3:0-61 9:2-8
 @ ./node_modules/babel-loader/lib/index.js??clonedRuleSet-4.use!./node_modules/vue-loader/lib/index.js??vue-loader-options!./src/App.vue?vue&type=script&lang=js& 21:0-58 61:17-28
 @ ./src/App.vue?vue&type=script&lang=js& 1:0-178 1:194-197 1:199-374 1:199-374
 @ ./src/App.vue 2:0-55 3:0-50 3:0-50 10:2-8
 @ ./src/main.js 2:0-28 24:13-16

ERROR in ./node_modules/@nextcloud/vue/dist/index.module.js 2:500715-500730
Module not found: Error: Can't resolve 'util' in '/home/pi/devNextcloud/appointments/node_modules/@nextcloud/vue/dist'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "util": require.resolve("util/") }'
        - install 'util'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "util": false }
 @ ./node_modules/babel-loader/lib/index.js??clonedRuleSet-4.use!./node_modules/vue-loader/lib/index.js??vue-loader-options!./src/components/ActionInputExt.vue?vue&type=script&lang=js& 1:0-47 4:19-32 6:11-24 10:6-41
 @ ./src/components/ActionInputExt.vue?vue&type=script&lang=js& 1:0-195 1:211-214 1:216-408 1:216-408
 @ ./src/components/ActionInputExt.vue 2:0-66 3:0-61 3:0-61 9:2-8
 @ ./node_modules/babel-loader/lib/index.js??clonedRuleSet-4.use!./node_modules/vue-loader/lib/index.js??vue-loader-options!./src/App.vue?vue&type=script&lang=js& 21:0-58 61:17-28
 @ ./src/App.vue?vue&type=script&lang=js& 1:0-178 1:194-197 1:199-374 1:199-374
 @ ./src/App.vue 2:0-55 3:0-50 3:0-50 10:2-8
 @ ./src/main.js 2:0-28 24:13-16

12 errors have detailed information that is not shown.
Use 'stats.errorDetails: true' resp. '--stats-error-details' to show it.

webpack 5.75.0 compiled with 12 errors in 59904 ms
```

**Comment by SergeyMosin** [comment]
Thanks for the report, I've locked "@nextcloud/vue" at version "~7.3.0" for now. I'll check v7.4.0 compatibility when I have more time

**Comment by serotonie** [comment]
> Thanks for the report, I've locked "@nextcloud/vue" at version "~7.3.0" for now. I'll check v7.4.0 compatibility when I have more time

You're welcome.
I'm sorry not to be able to help you more, that's my first time working with vue.js and webpack and adding the features needed for my very own use case is already a challenge in itself ☺️

Anyway, thanks a lot for your work on this awesome app !

Is there anywhere I could help you fund your much needed coffee (nothing incredible, I'm not rich my self and don't have a big budget, but something is better than nothing) ?

**Comment by nolens** [comment]
@serotonie 

Funding page can be found here: 
https://www.srgdev.com/gh-support/nextcloudapps

**Comment by SergeyMosin** [comment]
fixed in v1.14.11
